bundle/manifests/crunchy-bridge-operator.clusterserviceversion.yaml

```diff
@@ -432,4 +432,5 @@ spec:
   provider:
     name: Crunchy Data
     url: https://www.crunchydata.com
-  version: 0.0.2-dev
+  replaces: crunchy-bridge-operator.v0.0.2
```

Comment:
Are the values for this version-specific, or can it define replaces for a range? This change references v0.0.2, but v0.0.3 is the latest on our server.

Reply:
our issue is we didn't use 0.0.3 and that version doesn't have upgrade logic build into it, so clusters with 0.0.2 wouldn't upgrade to 0.0.4. that said, I'll look into implementing the `olm.skipRange` annotation for this... as it's the better approach should it work